MinecraftClient/Protocol/Handlers/Protocol18.cs

```diff
@@ -1230,5 +1340,42 @@ public bool SendUseItemPacket(int hand)
             catch (System.IO.IOException) { return false; }
             catch (ObjectDisposedException) { return false; }
         }
+
+        public bool SendPlayerBlockPlacement(int hand, Location location, int face, float CursorX, float CursorY, float CursorZ, bool insideBlock)
+        {
+            try
+            {
+                List<byte> packet = new List<byte>();
+                packet.AddRange(dataTypes.GetVarInt(hand));
+                packet.AddRange(dataTypes.GetLocation(location));
+                packet.AddRange(dataTypes.GetVarInt(face));
+                packet.AddRange(dataTypes.GetFloat(CursorX));
+                packet.AddRange(dataTypes.GetFloat(CursorY));
+                packet.AddRange(dataTypes.GetFloat(CursorZ));
+                packet.Add(Convert.ToByte(insideBlock ? 1 : 0));
+                SendPacket(PacketOutgoingType.PlayerBlockPlacement, packet);
+                return true;
+            }
+            catch (SocketException) { return false; }
+            catch (System.IO.IOException) { return false; }
+            catch (ObjectDisposedException) { return false; }
+        }
+
+        public bool SendHeldItemChange(short slot)
+        {
+            try
+            {
+                List<byte> packet = new List<byte>();
+                // short to byte (?
```

Comment:
Seems like the field should indeed be a short here:
https://wiki.vg/Protocol#Held_Item_Change_.28serverbound.29
Serverbound is packet send by client to the server (while clientbound uses a byte)
Would be handy to add `GetShort()` inside [DataTypes.cs](https://github.com/ORelio/Minecraft-Console-Client/blob/master/MinecraftClient/Protocol/Handlers/DataTypes.cs), then use `GetShort(slot)` here.